.appveyor.yml

```diff
@@ -193,19 +193,19 @@ notifications:
 
 
     Artifacts:
-    
+
```

Comment:
no there was white space on multiple lines in this file. so when i opened it in my IDE and made the changes then saved it. the IDE will automatically remove any unneeded white space.